packages/dashboard/src/customization/widgets/text/component.tsx

```diff
@@ -53,13 +54,17 @@ const TextWidgetComponent: React.FC<TextWidget> = (widget) => {
       return <StyledText {...widget} />;
     }
   } else {
-    if (isUrl) {
-      return <StyledTextArea isUrl {...props} />;
-    } else if (isEditing) {
-      return <StyledTextArea {...props} />;
-    } else {
-      return <EditableStyledText {...props} />;
-    }
+    return (
+      <WidgetTile widget={widget}>
+        {isUrl ? (
+          <StyledTextArea isUrl {...props} />
+        ) : isEditing ? (
+          <StyledTextArea {...props} />
+        ) : (
+          <EditableStyledText {...props} />
+        )}
```

Comment:
Avoid using multiple ternary operators in single statements, you can add tiny component or function and return like below:

```
const sampleFunc = ({isUrl, isEditing}) => {
if(isUrl) {
return <StyledTextArea isUrl {...props} />;
} else if(isEditing) {
return <StyledTextArea {...props} />;
} else {
return <EditableStyledText {...props} />;
}
```